R/scripts/hosp_run_geoid.R

```diff
@@ -0,0 +1,87 @@
+library(devtools)
```

Comment:
this should be deleted. Legacy version, but everything goes through hosp_run.R now

Reply:
deleted in dataseed_merge